Sub/Src/HSV Filtering/objectDistanceDetector.py

```diff
@@ -0,0 +1,86 @@
+class ObjectDistanceDetector:
```

Comment:
Not in correct directory. Should be in Sub/Src/ComputerVision. Also missing file header with description